app/models/trip.rb

```diff
@@ -10,8 +10,9 @@ class Trip < ApplicationRecord
                         :subscription_type,
                         :zip_code
   belongs_to :condition, optional: true
-  has_many :station_trips
-  has_many :stations, through: :trips
+
+  belongs_to :start_station, class_name: 'Station', optional: true
```

Comment:
Again, I think we should be able to refactor to remove the class_name. Leaving a comment here too so we don't forget it's in both places.